vllm/transformers_utils/config.py

```diff
@@ -41,6 +42,8 @@ def get_config(model: str,
         config = config_class.from_pretrained(model,
                                               revision=revision,
                                               code_revision=code_revision)
+    if rope_scaling is not None:
+        config.update({"rope_scaling": rope_scaling})
```

Comment:
Worth putting an info log here to saying "Updating rope_scaling from xxx to yyy"